packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -959,7 +1001,7 @@ class InspectorController extends DisposableController
     }
   }
 
-  void selectionChanged() {
+  void selectionChanged({bool notifyFlutterInspector = false}) {
```

Comment:
can you add dart doc describing what this param does? Is this the Flutter inspector in Devtools or the Flutter inspector in the flutter framework?

Reply:
Yes, done!